packages/calypso-e2e/src/lib/flows/start-import-flow.ts

```diff
@@ -128,6 +137,13 @@ export class StartImportFlow {
 		await this.page.click( selectors.wpContentOnlyContinueButton );
 	}
 
+	/**
+	 * Continue 'content only' WordPress migration on plans page.
+	 */
+	async contentOnlyWordPressPlansPage(): Promise< void > {
+		await this.page.click( selectors.wpPlansContentOnlyOptionButton );
```

Comment:
**question:non-blocking** - could you give a screenshot or link to the endpoint for this?

Reply:
Please see the screenshot below, it's a pre-migration page with plans information in it. Users can click the content-only option for free import here. 
![Screen Shot 2023-05-29 at 11 22 58 PM](https://github.com/Automattic/wp-calypso/assets/4074459/a217b4ec-26ea-43c7-85c6-d53c03cce8cb)